tracer/test/Datadog.Trace.TestHelpers/CustomTestFramework.cs

```diff
@@ -24,6 +24,22 @@ public CustomTestFramework(IMessageSink messageSink)
             : base(messageSink)
         {
             FluentAssertions.Formatting.Formatter.AddFormatter(new DiffPaneModelFormatter());
+
+            Task memoryDumpTask = null;
```

Comment:
I believe this is unused?
```suggestion
```

Reply:
Merging this myself to kick off another build, seeing as this is causing everything to fail